<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="host_group">
     <property name="title">
      <string>DFHack Host</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="host_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="host_address_label">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="host_port_label">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="host_address"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="host_port"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="host_autoconnect">
          <property name="text">
           <string>Auto-connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="autorefresh_group">
     <property name="title">
      <string>Auto-refresh</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="autorefresh_layout">
        <item row="1" column="0">
         <widget class="QLabel" name="autorefresh_interval_label">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="autorefresh_interval">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>600.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="autorefresh_enable">
          <property name="text">
           <string>Enabled</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advanced_group">
     <property name="title">
      <string>Advanced Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="advanced_layout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="use_native_process">
          <property name="text">
           <string>Use Native API</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="bypass_work_detail_protection">
          <property name="text">
           <string>Bypass work detail protection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gridview_group">
     <property name="title">
      <string>Grid views</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="gridview_layout">
        <item row="4" column="1">
         <widget class="QComboBox" name="rating_display_cb"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="ratin_display_label">
          <property name="text">
           <string>Rating display mode:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="gridview_perview_groups">
          <property name="text">
           <string>Per-view groups</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="gridview_perview_filters">
          <property name="text">
           <string>Per-view filters</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="gridview_sync_selection">
          <property name="text">
           <string>Synchronize selection between views</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reset_defaults_button">
     <property name="text">
      <string>Reset to defaults</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autorefresh_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>autorefresh_interval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
